config.json

```diff
@@ -640,7 +636,6 @@
         "conditionals",
         "lists",
         "loops",
-        "mathematics",
```

Comment:
I would argue that this exercise is still relevant to the `math` topic.

Reply:
After reviewing discussion in exercism/exercism.io#4110, I am OK with removing this topic from this exercise.